frontend/dashboard/pages/PageLayout/DashboardHeader/SubHeader/SubHeader.tsx

```diff
@@ -2,16 +2,30 @@ import React, { type ReactElement } from 'react';
 import { GiteaHeader } from 'app-shared/components/GiteaHeader';
 import { useSelectedContext } from '../../../../hooks/useSelectedContext';
 import { useOrgRepoName } from '../../../../hooks/useOrgRepoName';
+import { isOrg } from '../../../OrgContentLibrary/utils';
+import { useRepoStatusQuery } from 'app-shared/hooks/queries';
+import { StudioSpinner } from '@studio/components';
+import { useTranslation } from 'react-i18next';
 
-export type SubHeaderProps = {
-  hasRepoError?: boolean;
-};
-
-export const SubHeader = ({ hasRepoError }: SubHeaderProps): ReactElement => {
+export const SubHeader = (): ReactElement => {
```

Comment:
Nit: Since we explicitly return null in some scenarios, the return type should reflect that
```suggestion
export const SubHeader = (): ReactElement | null => {
```